general-superstaq/general_superstaq/service.py

```diff
@@ -94,6 +95,26 @@ def update_user_role(self, email: str, role: int) -> str:
             }
         )
 
+    def solve_qubo(
```

Comment:
I see this is being added here, but where is it being deleted from? Or did this function not exist before this PR?

Reply:
Correct, this method did not exist prior to this PR. This method simply calls the `submit_qubo` that is in `superstaq_client.py` (so it was a minor edit).

BTW, I have renamed `solve_qubo` to `submit_qubo`, as that is the name of the function our notebook calls.